general/administration/reverse-proxy.md

```diff
@@ -327,31 +327,41 @@ docker-compose.yml :
 version: '3'
 
 services:
-  proxy:
-    container_name: Traefik
-    image: traefik:alpine
-    command: --web --docker --docker.domain=${DOMAIN} --logLevel="INFO"
+  traefik:
+    container_name: traefik
+    hostname: traefik
+    image: traefik:traefik:v1.7.19
```

Comment:
Should probably not specify a specific version in our docs, as this is likely to change frequently.